man/systemd-hostnamed.service.xml

```diff
@@ -18,7 +18,7 @@
   <refnamediv>
     <refname>systemd-hostnamed.service</refname>
     <refname>systemd-hostnamed</refname>
-    <refpurpose>Host name bus mechanism</refpurpose>
+    <refpurpose>Daemon to control system hostname from programs</refpurpose>
```

Comment:
```suggestion
    <refpurpose>Daemon to control the system hostname from user programs</refpurpose>
```

Reply:
I'm not sure about "user" here. Technically speaking, all programs are user programs because they run in user space. But this API is suitable e.g. for NetworkManager or systemd-networkd, which are not something that the user runs directly.

Also, I think it's OK to skip the articles here, since it's a title, and we need it to be short.